### ISSUE TITLE
CalcMainActivity does not seem to work when added as a SF hook

### ISSUE THREAD

**Issue by ZH-Hristov** [thegrb93/StarfallEx#1626]
the hook
```
SF.hookAdd("CalcMainActivity", nil, nil, function(instance, args, ply, velocity)
	if args[1] then
		print(args[2], args[3])
		return args[2]
	end
end)
```
in-game test code
```
hook.add("CalcMainActivity", "crap", function(ply, vel)
    return -1, ply:lookupSequence("death_03")
end)
```
in GLua, the equivalent `lua_run_cl hook.Add("CalcMainActivity", "garb", function() return -1, 224 end)` works correctly

The player just T-poses no matter what is returned or how many args are returned

**Comment by thegrb93** [comment]
Looks like hook returns only accept one value currently https://github.com/thegrb93/StarfallEx/blob/master/lua/starfall/sflib.lua#L731

**Comment by ZH-Hristov** [comment]
Doesn't seem to work even if you're returning the first val (ACT only)

Tested with ply:getSequenceActivity(ply:lookupSequence(idle_ar2))

**Comment by thegrb93** [comment]
I forgot the return at each unpack. woops

**Comment by thegrb93** [comment]
Pushed a fix, try now?

**Comment by ZH-Hristov** [comment]
Still T-Posing. It's just this hook that seems to have a problem with SF for some god forsaken reason.

I tried TranslateActivity as well and that one works fine. I tried setting the gamemode hook override on the hookAdd to true but that didn't help.

**Comment by thegrb93** [comment]
@ZH-Hristov It's working fine for me while testing in singleplayer

**Comment by ZH-Hristov** [comment]
Ughh this is bizarre. With the same hookAdd snippet I posted?

**Comment by thegrb93** [comment]
Yeah, you can test it by doing `print(hook.Call("CalcMainActivity"))` I think

**Comment by ZH-Hristov** [comment]
@thegrb93
My mistake. I thought you'd pushed the change already and was trying it with the latest release code. Copied the sflib from the PR and it works correctly now. Thanks.
